Content.Server/Materials/MaterialReclaimerSystem.cs

```diff
@@ -157,6 +160,7 @@ public override bool TryFinishProcessItem(EntityUid uid, MaterialReclaimerCompon
             SpawnChemicalsFromComposition(uid, item, completion, false, component, xform);
             _body.GibBody(item, true);
             _appearance.SetData(uid, RecyclerVisuals.Bloody, true);
+            _adminLogger.Add(LogType.Gib, LogImpact.Extreme, $"{ToPrettyString(item):victim} was gibbed by {ToPrettyString(uid):entity} ");
```

Comment:
Have to move this above since the body doesn't exist anymore.

Reply:
Will do although I swear it worked just fine when I tested it.